main/src/main/scala/sbt/Defaults.scala

```diff
@@ -1778,7 +1778,7 @@ object Defaults extends BuildCommon {
         config,
         s.cacheStoreFactory,
         s.log,
-        sys.env.get("SOURCE_DATE_EPOCH").map(_.toLong * 1000).orElse(Some(0L))
+        sys.env.get("SOURCE_DATE_EPOCH").map(_.toLong * 1000).orElse(Some(315500400000L))
```

Comment:
Is this midnight in JST? 
Also given that Bazel uses 2010-01-01, maybe we should unify to that timestamp too, which is `1262304000000L`.

Reply:
Ah, this is JST. I should have used UTC midnight. Anyway, using 1262304000000L sounds good!

Reply:
Updated the pull request.